Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2435,6 +2435,32 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
 
 // TODO: Consider making the return type be generic over `FloatingPoint` types
 // so that `self`'s scalar type can be any `Numeric` type.
+extension Tensor {
```

Comment:
Please move this newly added internal extension above the `TODO` comment from above. That `TODO` comment belongs on the `public extension Tensor where Scalar: TensorFlowFloatingPoint` extension below.

```swift
// TODO: Consider making the return type be generic over `FloatingPoint` types
// so that `self`'s scalar type can be any `Numeric` type.
```